basyx/aas/adapter/xml/xml_deserialization.py

```diff
@@ -692,8 +692,10 @@ def construct_extension(cls, element: etree.Element, object_class=model.Extensio
         value = _get_text_or_none(element.find(NS_AAS + "value"))
         if value is not None:
             extension.value = model.datatypes.from_xsd(value, extension.value_type)
-        extension.refers_to = _failsafe_construct_multiple(element.findall(NS_AAS + "refersTo"),
-                                                           cls._construct_referable_reference, cls.failsafe)
+        extension.refers_to = _failsafe_construct_multiple(
+            element.find(NS_AAS + "refersTo").findall(NS_AAS + "reference"),
+            cls._construct_referable_reference, cls.failsafe
+        )
```

Comment:
`element.find()` can be `None`, which may cause a crash here.
```suggestion
        refers_to = element.find(NS_AAS + "refersTo")
        if refers_to is not None:
            for ref in _child_construct_mutliple(refers_to, NS_AAS + "reference", cls._construct_referable_reference,
                                                 cls.failsafe):
                extension.refers_to.add(ref)
```

Reply:
Use `extension.refers_to.add()` or `extension.refers_to.append()` depending on whether you decide on using a `Set` or a `List` for the `refers_to` attribute.

Reply:
Fixed in daa770f